### PR TITLE
tier1, add ovs-cni e2e-functests script

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
This script adds ovs-cni e2e functests on a cluster with CNAO latest changes.

This PR can only be merged after:
1. ovs-cni issues a new release with functests [PR](https://github.com/kubevirt/ovs-cni/pull/124) is mereged
2. ovs-cni is bumped
3. lane added in project infra [PR](https://github.com/kubevirt/project-infra/pull/582)

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RamLavi** [comment]
/hold

until PR #635 is merged

**Comment by RamLavi** [comment]
/unhold 
#635 got merged

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
